src/app/zap_cluster_list.json

```diff
@@ -131,28 +135,48 @@
         "LAUNDRY_DRYER_CONTROLS_CLUSTER": [],
         "WIFI_NETWORK_DIAGNOSTICS_CLUSTER": [],
         "WINDOW_COVERING_CLUSTER": [],
+        "METER_INDENTIFICATION_CLUSTER": [],
```

Comment:
Why all the spurious changes to this file?  And this bit is clearly wrong....

I would not have expected any changes to this file at all until we are adding implementation for this cluster.

Reply:
It is result of zap tool.
This PR has not manual work.
- alchemy converts *.adoc to *.xml
- zap tool generates other from *.xml

Reply:
It's possible alchemy makes modifications to this file, but where exactly did the string "METER_INDENTIFICATION_CLUSTER" come from?  Running alchemy on top of an already-modified tree can lead to weird issues like this.  I am guessing you hand-edited this json file, then ran alchemy on top of those changes.

Reply:
Hm, than I am going to regen all with a clean slate

Reply:
This change is result of the alchemy tool - I checked

Reply:
> This change is result of the alchemy tool - I checked

I am 100% sure that "METER_INDENTIFICATION_CLUSTER" was not the tool.  And that part is gone in the PR now, which is good.